### DISCUSSION TITLE
Difficult to understand how to use honeypot tool.

### DISCUSSION THREAD

**Discussion by 0MAL** [mosparo/mosparo#206] [Ideas]
As title says, additional explanation is required, I think. I tried to use honeypot with html code by manually creating and including invisible fields, and it fails, finally I understand I don't need to prepare honeypot field by myself. All I have to do is  just activate honeypot and input dummy field name for it in mosparo's config page.

**Comment by zepich** [comment]
Hi @0MAL 

Thank you very much for your question.

The honeypot security feature is an additional simple feature to help catch spam. If you enable it, you have to specify a field name. mosparo then automatically adds an invisible form field to your form. You don't have to add any other honeypot field (you can, of course, but you don't have to), and mosparo does everything else for you regarding the honeypot field. If the spam bot fills out the honeypot field, mosparo will automatically reject the submission.

If you add your own honeypot fields to the form, then you must ensure that the honeypot fields are also verified by mosparo during the verification process. Because your own honeypot fields are normal fields for mosparo, mosparo validates these fields and expects their verification.

mosparo allows you to set the name of the honeypot field because we don't know which fields you already have in your form, and the honeypot field should have a natural name.

Do you think we should adjust the explanation in the user interface or the documentation?

Thank you very much.

Kind regards,

zepich

**Comment by 0MAL** [comment]
I think adding a bit more explanation is helpful.

**Reply by zepich** [reply]
Okay, perfect, thank you very much for your feedback!

We will optimize the explanation in the tool and in the documentation.
